pkg/sdk/go/groups.go

```diff
@@ -187,7 +187,7 @@ func (sdk mgSDK) RemoveUserFromGroup(groupID string, req UsersRelationRequest, t
 }
 
 func (sdk mgSDK) ListGroupUsers(groupID string, pm PageMetadata, token string) (UsersPage, errors.SDKError) {
```

Comment:
This should be in `users.go`. Cross check other SDK:
- for example `/things/channels/channelID` should be in things.go
- `/channels/users/userID` should be in `channels.go`

Reply:
There is no `/things/channels/channelID` endpoint. We have `/things/channels/thingID` which lists channels by thingID, and I think this one should remain in the channels.go file.

What do you think @rodneyosodo ?

Reply:
There is https://github.com/WashingtonKK/magistrala/blob/69b579199035c44acc352e3f588f0eba2ddac163/things/api/http/clients.go#L119